Storage/SQL/BrowserDB.swift

```diff
@@ -138,10 +139,9 @@ public class BrowserDB {
         return success
     }
 
-    typealias CursorCallback = (connection: SQLiteDBConnection, inout err: NSError?) -> Cursor
     typealias IntCallback = (connection: SQLiteDBConnection, inout err: NSError?) -> Int
 
-    func insert(inout err: NSError?, callback: IntCallback) -> Int {
+    func withWritableConnection(inout err: NSError?, callback: IntCallback) -> Int {
```

Comment:
You might as well just use withConnection, since this readable/writable stuff is basically lies now (i.e. we only have one connection and its always read/write/create).